src/test/java/com/hadoop/compression/lzo/TestDistLzoIndexerJobName.java

```diff
@@ -0,0 +1,108 @@
+package com.hadoop.compression.lzo;
+
+import junit.framework.TestCase;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.mapreduce.Job;
+
+public class TestDistLzoIndexerJobName extends TestCase {
+
+  public void testDefaultName() throws Exception {
+    String[] args = new String[]{
+        "hdfs://cluster/user/test/output/file-m-00000.lzo",
+    };
+
+    Job job = new Job(new Configuration(false));
+    DistributedLzoIndexer.setJobName(job, args);
+
+    String expected = "Distributed Lzo Indexer [hdfs://cluster/user/test/output/file-m-00000.lzo]";
+
+    assertEquals(expected, job.getJobName());
+  }
+
+  public void testCustomeName() throws Exception {
+    String[] args = new String[]{
+        "ignored",
+    };
+    String customName = "-<Custom Job Name>-";
+
+    Configuration conf = new Configuration(false);
+    conf.set("lzo.indexer.distributed.job.name", customName);
+    Job job = new Job(conf);
+    DistributedLzoIndexer.setJobName(job, args);
+
+    assertEquals(customName, job.getJobName());
+  }
+
+  public void testCustomeNameTruncation() throws Exception {
+    String[] args = new String[]{
+        "ignored",
+    };
+
+    Configuration conf = new Configuration(false);
+    conf.set("lzo.indexer.distributed.job.name", "123456789");
+    conf.setInt("lzo.indexer.distributed.job.name.max.length", 5);
+    Job job = new Job(conf);
+    DistributedLzoIndexer.setJobName(job, args);
+
+    assertEquals("12345...", job.getJobName());
+  }
+
+  public void testDefaultLengthTruncation() throws Exception {
```

Comment:
I don't really think it's important to test the default length. It should be sufficient between the custom name test and custom length test IMO. I'm OK with dropping this test. The fact that this test needs to rely on the default value (200) implicitly is also bit of a drawback (should the default change in the future).